pkg/validations/createvalidations/preflightvalidations.go

```diff
@@ -105,6 +105,13 @@ func (v *CreateValidations) PreflightValidations(ctx context.Context) []validati
 					Err:         validations.ValidateManagementClusterEksaVersion(ctx, k, v.Opts.ManagementCluster, v.Opts.Spec),
 				}
 			},
+			func() *validations.ValidationResult {
+				return &validations.ValidationResult{
+					Name:        "validate eksa release components exist on management cluster",
+					Remediation: fmt.Sprintf("check eksaVersion field for workload cluster %s matches release previously used to upgrade maangement cluster %s", v.Opts.WorkloadCluster.Name, v.Opts.ManagementCluster.Name),
```

Comment:
nit: this message is a bit confusing IMO

What do you think about saying something like "make sure you use one of the available versions and providing the kubectl command to read the list of eksa versions?

Reply:
I agree. Let me know if the new remediation message is still confusing.